driver-sync/src/test/functional/com/mongodb/client/unified/ContextElement.java

```diff
@@ -83,21 +87,41 @@ public String toString() {
         }
     }
 
-    private static class OperationContextElement extends ContextElement {
+    private static class StartedOperationContextElement extends ContextElement {
+        private final BsonDocument operation;
+        private final int index;
+
+        StartedOperationContextElement(final BsonDocument operation, final int index) {
+            this.operation = operation;
+            this.index = index;
+        }
+
+        public String toString() {
+            return "Started Operation Result Context: " + "\n"
+                    + "   Operation:\n"
+                    + operation.toJson(JsonWriterSettings.builder().indent(true).build()) + "\n"
+                    + "   Operation index: " + index + "\n";
+        }
+    }
+
+    private static class CompletedOperationContextElement extends ContextElement {
         private final BsonDocument operation;
         private final OperationResult result;
+        private final int index;
 
-        OperationContextElement(final BsonDocument operation, final OperationResult result) {
+        CompletedOperationContextElement(final BsonDocument operation, final OperationResult result, final int index) {
             this.operation = operation;
             this.result = result;
+            this.index = index;
         }
 
         public String toString() {
-            return "Operation Result Context: " + "\n"
+            return "Completed Operation Result Context: " + "\n"
```

Comment:
It appears that this should be either "Completed Operation Context" or "Operation Result Context" (the original text). However, "Completed Operation Context" is more consistent with the class name `CompletedOperationContextElement`.

Reply:
Done